modules/ROOT/pages/sap/sap-connector-reference-42.adoc

```diff
@@ -0,0 +1,836 @@
+
+=== Inbound Associated Sources
+
+* <<DocumentSource>> 
+* <<FunctionSource>> 
+
+---
+[[outbound-config]]
+== Outbound Configuration
+
+Default configuration for SAP extension outbound elements.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<outbound-config_certificate, Certificate>> 
+* <<outbound-config_simple-connection-provider, Simple Connection Provider>> 
+ | The connection types to provide to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when it sees fit. |  | 
+| Encoding a| String |  |  | 
+| Default Idocument Version a| String |  |  0 | 
+| Disable Function Template Cache Flag a| Boolean |  |  `false` | 
+| Evaluate Response Flag a| Boolean |  |  `false` | 
+| Log Trace Flag a| Boolean |  |  `false` | 
+| Use SAP Date Time Format a| Boolean a|  This parameter was added to honor the Mule 3 format for returning DATE and TIME values. Used in the `JCoToBusinessObjectParser` class when parsing a JCoField.
+
+* `true` - Return the DATE and TIME type objects as strings formatted as in SAP. The DATE is in the  `yyyy-MM-dd` format, and TIME is in the `HH:mm:ss` format.
+* `false` (default) - Return the DATE and TIME values. |  `false` | 
+|===
+
+=== Outbound Connection Types
+
+[[outbound-config_certificate]]
+==== Certificate
+
+
+Connection provider that relies on the usage of a X509 certificate to authenticate the user.
+
+
+===== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| SAP system number a| String |  The two-digit system number (sysnr) of the SAP system. |  | x
+| SAP client ID a| String |  Three-digit client number. Preserve leading zeros if they appear in the number. |  | x
+| Destination Name a| String |  Identifier of the destination generated by this connector. If none is set, then a random value will be created. |  | 
+| Login language a| String |  Code of the language used by the connector. The language determines the code page used for communicating between SAP Adapter and the application server. |  EN | 
+| Extended Properties a| Object |  |  | 
+| X.509 Certificate a| String |  Path to the X.509 certificate. This is required when the connector is working as a client. |  | 
+| Application server host a| String |  The host of the SAP application server. |  | 
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage. |  | 
+| Enable RFC tracing on server a| Boolean |  Enable or disable RFC trace on the server. |  false | 
```

Comment:
"on the server" If you make this change, you can search for more occurrences of this.